docs/DeveloperGuide.md

```diff
@@ -270,20 +270,19 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                   | I want to …​              | So that I can…​                                                         |
-|----------|-------------------------------------------| ------------------------- |-------------------------------------------------------------------------|
-| `* * *`  | new user                                  | see usage instructions    | refer to instructions when I forget how to use the App                  |
-| `* * *`  | user                                      | add a new person          |                                                                         |
-| `* * *`  | user                                      | delete a person           | remove entries that I no longer need                                    |
-| `* * *`  | user                                      | find a person by name     | locate details of persons without having to go through the entire list  |
-| `* *`    | user                                      | hide private contact details | minimize chance of someone else seeing them by accident                 |
-| `*`      | user with many persons in the address book | sort persons by name      | locate a person easily                                                  |
-| `* * *`  | home baker with multiple orders           | mark the orders as complete or incomplete | know which orders i have fulfilled or not                               |
-| `* * *`  | home baker that has multiple customers    | clear all my customers    | I can quickly remove demo info or restart my bakery data                |
-| `* * *`  | home baker that has multiple orders	     | look at my orders | I can access the attributes for different orders and see when it is due |
```

Comment:
This information has been removed, please add it back into the table if it has been accidentally deleted!

Reply:
@Geralddtan I think it looks fine,
Comparing the two, it looks like @punnyhuimin edited the user stories appropriately